src/main/java/pl/javastart/task/Agreement.java

```diff
@@ -0,0 +1,60 @@
+package pl.javastart.task;
+
+abstract class Agreement {
+
+    private int smsCount = 0;
+    private int mmsCount = 0;
+    private int secCount = 0;
+    private double accountState = 0;
```

Comment:
Czy to jest na pewno wspólne dla wszystkich umów?
W Subscription to pole zawsze będzie miało wartość 0